integration/simulation/devnetwork/dev_network.go

```diff
@@ -82,6 +84,10 @@ func (s *InMemDevNetwork) ValidatorRPCAddress(idx int) string {
 	return val.HostRPCAddress()
 }
 
+func (s *InMemDevNetwork) GetL1Client() (ethadapter.EthClient, error) {
+	return s.l1Network.GetClient(0), nil
```

Comment:
maybe add a comment here that this could be more elaborate if we start shutting down the l1 nodes to test various things?